projects/optic/src/commands/capture/sources/proxy.ts

```diff
@@ -56,19 +63,10 @@ export class ProxyInteractions {
       );
       throw new UserError();
     }
-    if (targetHost.includes('/')) {
-      // accept urls to be passed in rather than pure hosts
-      targetHost = host;
-    }
```

Comment:
Uhm I'm confused what this is trying to do, but this would always return true, since new URL needs a `protocol` (`http://` or `https://`)